docs/site/OpenAPI-generator.md

```diff
@@ -429,21 +428,6 @@ export class Test2DataSource extends juggler.DataSource
   ) {
     super(dsConfig);
   }
-
-  /**
-   * Start the datasource when application is started
-   */
-  start(): ValueOrPromise<void> {
-    // Add your logic here to be invoked when the application is started
-  }
-
-  /**
-   * Disconnect the datasource when application is stopped. This allows the
-   * application to be shut down gracefully.
-   */
-  stop(): ValueOrPromise<void> {
-    return super.disconnect();
-  }
```

Comment:
Should we keep this part if we are adding a comment at L415?

Reply:
My main intention and motivation for this work is to simplify the DataSource classes scaffolded in LB4 applications and avoid boilerplate start/stop methods. That's why I added `stop` method to `juggler.DataSource` inside loopback-datasource-juggler, so that LB4 applications don't have to implement that method themselves. See https://github.com/strongloop/loopback-datasource-juggler/pull/1835